nixos/modules/testing/test-instrumentation.nix

```diff
@@ -118,8 +115,8 @@ in
     boot.consoleLogLevel = 7;
 
     # Prevent tests from accessing the Internet.
-    networking.defaultGateway = mkOverride 150 "";
-    networking.nameservers = mkOverride 150 [ ];
+    networking.defaultGateway = lib.mkOverride 150 "";
+    networking.nameservers = lib.mkOverride 150 [ ];
 
     system.requiredKernelConfig = with config.lib.kernelConfig; [
```

Comment:
This is an example I will not touch. It is ugly but way less injustifiable than `meta = with lib; { maintainers = with pkgs.lib; . . .`